README.md

```diff
@@ -1 +1,106 @@
-iRODS HTTP API: https://github.com/irods/irods_client_http_api
+# iRODS Java HTTP API Wrapper
+
+Java client library designed and developed to wrap the new iRODS HTTP API. It provides java classes to cleanly interact 
+with all the functionality of the HTTP API and comes with tests and packaging.
+
+## How to Use?
+### Client
+Create a new `IrodsHttpClient` instance and pass in the base URL. 
+```java
+IrodsHttpClient client = new IrodsHttpClient("http://localhost:<port>/irods-http-api/<version>");
+```
+
+`<version>` must match the format, X.Y.Z, where X is the major version number, Y is the minor version number, and Z is the patch version number.
+
+You can only access the information endpoint without authenticating. All other operations require a token.
+
+### How to Use Operations
+All methods can be called by using a method chaining approach following this layout: 
+```java
+client.<endpoint>.<operation>(parameters);
+```
+On the client object, you chain the endpoint name (for example, collections), and then chain the operation name 
+(createCollection). 
+
+### What is being returned
+All methods return a `Response` object, which holds the HTTP status code and the body of the response (as JSON). Can be 
+retrieved by using `.getHttpStatusCode()` and `.getBody()`, respectively. You can parse the JSON response body using your 
+preferred JSON parsing library or method.
```

Comment:
Let's make this a comment in the usage example.

Reply:
Done.